CONTRIBUTING.md

```diff
@@ -33,9 +33,16 @@ will try to clarify it.
 
 * Add tests relevant to the fixed bug or new feature.
 
-* Add a [DCO](https://developercertificate.org/) / `Signed-off-by` line in any commit message.
+* Commit messages must contain both a [Developer Certificate of Origin](https://developercertificate.org/) / `Signed-off-by` line and a `RELEASE_NOTES=` entry, for example:
+
+      One line description of commit
+
+      More detailed description of commit, if needed.
+
+      RELEASE_NOTES=Description for release notes, or n/a if trivial.
```

Comment:
Please list the typical tags we are using:
[DOCUMENTATION], [BUGFIX], [UX], [CLEANUP], [BREAKING], [DEPRECATION], [ENHANCEMENT], [FEATURE], [TESTING]

Reply:
Done in #1644.